pages/common/multitail.md

```diff
@@ -0,0 +1,20 @@
+# multitail
+
+> Extension of tail.
+> More information: <https://www.vanheusden.com/multitail/examples.php>.
+
+- Tail all files matching a pattern in a single stream:
+
+`multitail -Q 1 '{{pattern}}'`
+
+- Tail all files in a directory in a single stream:
+
+`multitail -Q 1 '{{directory}}/*'`
+
+- Automatically add new files to a window:
+
+`multitail -Q {{pattern}}`
+
+- Show 5 logfiles while merging 2 and put them in 2 columns with only one in the left column:
```

Comment:
This is lifted verbatim from the home page and could be improved IMO. It is not clear to me what this is trying to do. So first of all, it shows 5 files, which means the 4 input file and the output file. And the merged file is the combination of 2 files ? Which 2 ? What does putting the output in 2 columns mean ? Which one is in the left column ?

Sorry but I don't understand any of it.

Reply:
Ah, I must have missed this! We should open a new PR to improve this.